ansible/configs/ocp4-cluster/lifecycle_hook_post_start.yml

```diff
@@ -79,3 +79,43 @@
       when: r_new_csrs.resources | length > 0
       command: "oc adm certificate approve {{ item.metadata.name }}"
       loop: "{{ r_new_csrs.resources }}"
+
+    # Get OpenShift Version to determine if we need additional recovery actions
+    - name: Get OpenShift ClusterVersion
+      k8s_facts:
+        api_version: config.openshift.io/v1
+        kind: ClusterVersion
+        name: version
+      register: r_cluster_version
+    - name: Set ocp4_cluster_version fact
```

Comment:
please add line above to maintain formatting throughout?

Reply:
If you insist. I like to keep things together that belong together.

Reply:
I could have sworn we discussed this in the code quality contributors guide, but I don't see it now hmmmmm. Still, but that logic, it is hard to draw the line between what belongs together and what doesn't...kind of subjective and this makes it more readable IMO.